### ISSUE TITLE
Failing pester tests

### ISSUE THREAD

**Issue by cstewart87** [PowerShellOrg/cWebAdministration#19]
I checked out the smurawski/adding_tests branch to test out Pester by running the unit tests that have been added.

All of the existing tests failed with the error message "Please ensure that WebAdministration module is installed."  Isn't WebAdministration a reference to this module?  If so, should this reference be changed to "xWebAdministration"?  Or is this another module that needs to be installed prior to running tests?

I believe the message is issued [here](https://github.com/PowerShellOrg/cWebAdministration/blob/smurawski/adding_tests/DSCResources/PSHOrg_cAppPool/PSHOrg_cAppPool.psm1#L796-L799)

**Comment by smurawski** [comment]
No, it is the webadministration module (which is present when IIS is installed).. Can you paste the full error (or put it in a gist)?

**Comment by cstewart87** [comment]
Thanks for the note, @smurawski.

That's my issue, IIS is not present on my development workstation.  I'm new to Pester, but I come from a Chef/Ruby background with ChefSpec/Rspec experience.  Are Pester unit tests expected to be run on a target node?  Or is it common to run them on a development workstation and it's just a case that I'm missing some dependent "libraries"?

```
PS C:\Users\cstewart\Documents\cWebAdministration> Invoke-Pester .\test\unit
Describing Test-Target resource responds when
   Context there is no existing app pool
    [-] returns false 766ms
      Please ensure that WebAdministration module is installed.
      at line: 798 in C:\Users\cstewart\Documents\cWebAdministration\DscResources\PSHOrg_cAppPool\PSHOrg_cAppPool.psm1
Describing Test-Targetresource responds when
   Context there is an existing app pool and everything matches the defaults
    [-] returns true 192ms
      Please ensure that WebAdministration module is installed.
      at line: 798 in C:\Users\cstewart\Documents\cWebAdministration\DscResources\PSHOrg_cAppPool\PSHOrg_cAppPool.psm1
   Context there is an existing app pool and the managed runtime should be v2.0
    [-] returns false 58ms
      Please ensure that WebAdministration module is installed.
      at line: 798 in C:\Users\cstewart\Documents\cWebAdministration\DscResources\PSHOrg_cAppPool\PSHOrg_cAppPool.psm1
   Context there is an existing app pool and the managed runtime should be v2.0 and the verbose stream is enabled
    [-] Error occurred in Context block 150ms
      Please ensure that WebAdministration module is installed.
      At C:\Users\cstewart\Documents\cWebAdministration\DscResources\PSHOrg_cAppPool\PSHOrg_cAppPool.psm1:798 char:9
Describing Set-TargetResource responds when
Tests completed in 1.17s
Passed: 0 Failed: 4 Skipped: 0 Pending: 0
```

**Comment by cstewart87** [comment]
I just wanted to add that after turning on the IIS feature in my workstation, the unit tests passed.  Thanks for the support!

**Comment by smurawski** [comment]
I want to see how I can remove that dependency.. I realized that my dev environment where I ran the tests had IIS after you filed the issue.  The unit tests should be run on your workstation.  Think somewhere between rspec and chefspec, then the integration tests are designed to be used with test-kitchen (I've got a dsc provisioner plugin for test-kitchen out - kitchen-dsc).

**Comment by cstewart87** [comment]
I've seen the Test Kitchen/DSC work that's been done recently, that's fantastic!  That's also how I came across these sample tests.

Thanks again for the support
